core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/atom/text/TextDisplayMedium.kt

```diff
@@ -39,3 +46,43 @@ fun TextDisplayMedium(
         style = MainTheme.typography.displayMedium,
     )
 }
+
+@Composable
+fun AutoResizeableTextDisplayMedium(
+    text: String,
+    modifier: Modifier = Modifier,
+    color: Color = Color.Unspecified,
+    textAlign: TextAlign? = null,
+    style: TextStyle = MainTheme.typography.displayMedium,
```

Comment:
Please remove the `style` parameter. 

If we need this functionality for another text style we can then figure out how best to generalize (and where to move) this code.

Reply:
Done!